hw_13/GenericStack.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace hw_13
+{
+    class GenericStack<T>
+    {
+        public readonly int capacity;
```

Comment:
generally fields are private. if we need to have an accessibility out - we declare a Property